package.json

```diff
@@ -230,7 +230,7 @@
     "usocket": "_EXCLUDED_"
   },
   "optionalDependencies": {
-    "macos-notification-state": "github:ferdium/macos-notification-state#130bb2602795b65ae2dea74d823027b185eaf0ae",
+    "macos-notification-state": "github:ferdium/macos-notification-state#e0465373b0aed12ea4ce30522e82aaa8706cdf40",
```

Comment:
This doesn’t use the original module now but still the fork?

Reply:
Yes, I forgot that we need to use a patch to compile the module, so I made the change into our fork (just updated the title and description so that they reflects that change)